test/unit/assorted/server_selection_logic_spec_utils.ts

```diff
@@ -0,0 +1,154 @@
+import { Document, EJSON } from 'bson';
+import { expect } from 'chai';
+import { readdirSync, readFileSync, statSync } from 'fs';
+import { basename, extname, join } from 'path';
+
+import {
+  ReadPreference,
+  ReadPreferenceMode,
+  ReadPreferenceOptions
+} from '../../../src/read_preference';
+import { ServerType, TopologyType } from '../../../src/sdam/common';
+import { ServerDescription, TagSet } from '../../../src/sdam/server_description';
+import * as ServerSelectors from '../../../src/sdam/server_selection';
```

Comment:
Can we import only the selectors we're testing? it makes it easy to grok the coverage

Reply:
Done!